adserver/models.py

```diff
@@ -686,7 +686,11 @@ class Flight(TimeStampedModel, IndestructibleModel):
     HIGHEST_PRIORITY_MULTIPLIER = 1000000
     LOWEST_PRIORITY_MULTIPLIER = 1
 
-    DEFAULT_INTERVAL = 60 * 60 * 24  # 24 hours
+    # The pacing interval in seconds used to pace flights evenly.
+    # For example, with a 1000 click flight over 10 days,
+    # there will be 10 intervals (each 1 day) and we'll aim for 100 clicks per day
+    # Shorter intervals (eg. an hour) spread flights more evenly across geographies
+    DEFAULT_PACING_INTERVAL = 60 * 60 * 24  # 24 hours
```

Comment:
I think we probably want this to be a setting still? but maybe not?

Reply:
I think it will require a migration since the default is used as a DB default.